ximage/patch/slices.py

```diff
@@ -225,20 +233,22 @@ def get_slice_around_index(index, size, min_start=0, max_stop=np.inf):
     Returns
     -------
     slice
-        The new slice object with a size of at least min_size and respecting the left and right bounds.
+        A slice object with the desired size and respecting the left and right bounds.
 
```

Comment:
Should we add that the original slice is returned if the slice is already larger than `size `?